packages/client/mutations/UpsertTeamPromptResponseMutation.ts

```diff
@@ -1,32 +1,18 @@
 import graphql from 'babel-plugin-relay/macro'
 import {commitMutation} from 'react-relay'
-import {StandardMutation} from '../types/relayMutations'
-import clientTempId from '../utils/relay/clientTempId'
-import createProxyRecord from '../utils/relay/createProxyRecord'
+import {UpsertTeamPromptResponseMutation_meeting} from '~/__generated__/UpsertTeamPromptResponseMutation_meeting.graphql'
+import {SharedUpdater, StandardMutation} from '../types/relayMutations'
 import {UpsertTeamPromptResponseMutation as TUpsertTeamPromptResponseMutation} from '../__generated__/UpsertTeamPromptResponseMutation.graphql'
 
 graphql`
   fragment UpsertTeamPromptResponseMutation_meeting on UpsertTeamPromptResponseSuccess {
+    meetingId
     teamPromptResponse {
       id
+      userId
       content
       plaintextContent
     }
-    meeting {
-      ... on TeamPromptMeeting {
-        id
-        createdAt
-        phases {
-          ... on TeamPromptResponsesPhase {
-            stages {
-              ... on TeamPromptResponseStage {
-                ...TeamPromptResponseCard_stage
-              }
-            }
-          }
-        }
-      }
-    }
```

Comment:
we don't actually need that, updater will just insert the response for the stage related to a given user